nav2_dwb_controller/dwb_core/src/publisher.cpp

```diff
@@ -197,9 +207,6 @@ DWBPublisher::publishTrajectories(const dwb_msgs::msg::LocalPlanEvaluation & res
     }
     ma.markers.push_back(m);
   }
-  addDeleteMarkers(ma, currentValidId, validNamespace);
```

Comment:
Why removing the addDeleteMarkers?

Reply:
Instead of these new update utilizes `visualization_msgs::msg::Marker::lifetime`.